pyaedt/generic/settings.py

```diff
@@ -0,0 +1,444 @@
+            "ANSYSEM_FEATURE_S432616_LAYOUT_COMPONENT_IN_3D_ENABLE": "1",
+            "ANSYSEM_FEATURE_F545177_ECAD_INTEGRATION_WITH_APHI_ENABLE": "1",
+            "ANSYSEM_FEATURE_F650636_MECH_LAYOUT_COMPONENT_ENABLE": "1",
+        }
+        if is_linux:
+            self._aedt_environment_variables["ANS_NODEPCHECK"] = "1"
+        self._desktop_launch_timeout = 90
+        # self._aedt_process_id = None
+        # self._is_student = False
+        self._number_of_grpc_api_retries = 3
+        self._retry_n_times_time_interval = 0.1
+
+    @property
+    def retry_n_times_time_interval(self):
+        """Time interval between the retries by the ``_retry_n_times`` method."""
+        return self._retry_n_times_time_interval
+
+    @retry_n_times_time_interval.setter
+    def retry_n_times_time_interval(self, value):
+        self._retry_n_times_time_interval = float(value)
+
+    @property
+    def number_of_grpc_api_retries(self):
+        """Number of gRPC API retries. The default is ``3``."""
+        return self._number_of_grpc_api_retries
+
+    @number_of_grpc_api_retries.setter
+    def number_of_grpc_api_retries(self, value):
+        self._number_of_grpc_api_retries = int(value)
+
+    # @property
+    # def aedt_process_id(self):
+    #     """ID of the desktop process. The default is ``None``."""
+    #     return self._aedt_process_id
+    #
+    # @aedt_process_id.setter
+    # def aedt_process_id(self, value):
+    #     self._aedt_process_id = int(value)
+    #
+    # @property
+    # def is_student(self):
+    #     """Flag indicating whether the desktop process is set to the student version."""
+    #     return self._is_student
+    #
+    # @is_student.setter
+    # def is_student(self, value):
+    #     self._is_student = value
+
+    @property
+    def desktop_launch_timeout(self):
+        """Timeout in seconds for trying to launch AEDT. The default is ``90`` seconds."""
+        return self._desktop_launch_timeout
+
+    @desktop_launch_timeout.setter
+    def desktop_launch_timeout(self, value):
+        self._desktop_launch_timeout = int(value)
+
+    @property
+    def aedt_environment_variables(self):
+        """Environment variables that are set before launching a new AEDT session,
+        including those that enable the beta features."""
+        return self._aedt_environment_variables
+
+    @aedt_environment_variables.setter
+    def aedt_environment_variables(self, value):
+        self._aedt_environment_variables = value
+
+    @property
+    def lsf_queue(self):
+        """LSF queue name. This attribute is valid only on Linux
+        systems running LSF Scheduler."""
+        return self._lsf_queue
+
+    @lsf_queue.setter
+    def lsf_queue(self, value):
+        self._lsf_queue = value
+
+    @property
+    def use_lsf_scheduler(self):
+        """Whether to use LSF Scheduler. This attribute is valid only on Linux
+        systems running LSF Scheduler."""
```

Comment:
```suggestion
        """Flag indicating if LSF Scheduler is being used. This attribute is valid only on Linux
        systems running LSF Scheduler."""
```
In the case of a property, the description should be a noun--and aren't you basically querying here to see if LSF Scheduler is turned on for use?